cmsdb/processes/higgs.py

```diff
@@ -27,36 +27,56 @@
 # Single Higgs
 #
 
+# preliminary Higgs cross sections at 13.6 TeV taken from here:
+# https://cds.cern.ch/record/2886099
+# TODO: h xsecs at 13 TeV
+
 h = Process(
     name="h",
     id=10000,
-    xsecs={13: Number(0.1)},  # TODO
+    xsecs={
+        13: Number(0.1),  # TODO
+        13.6: Number(0.1),  # TODO
+    },
 )
 
 h_ggf = h.add_process(
     name="h_ggf",
     id=11000,
     label=r"$H_{ggf}$",
-    xsecs={13: Number(0.1)},  # TODO
+    xsecs={
+        13: Number(0.1),  # TODO
+        13.6: Number(56.11, {
+            "pdf": 0.021j,
+            "th": (0.046j, 0.067j),
+            "th_gaussian": 0.040,
```

Comment:
```suggestion
        13.6: Number(52.23, {  # value for mH=125 GeV
            "pdf": 0.032j,
            "th": (0.046j, 0.067j),
            "th_gaussian": 0.039,
```

this is the cross section for mH=120 GeV. It is preferable to use the one for mH= 125 GeV, around 52.17 pb. You should also check the uncertainties, the pdf uncertainty I'm seeing is 0.032 for example. I would also a comment to say for which Higgs mass the value was taken (for bookkeeping)